apiary.apib

```diff
@@ -4294,7 +4294,89 @@ Customer activity log can be fetched using:
     [1091][]
     
 
-        
+
+
+## Delete Customer [/customer/id/{customerId}{?api_key}]
+
++ Parameters
+
+    + customerId (string, required) ... the Customer ID
+    + api_key (string, required) ... the API key
+
+### Customer Delete [DELETE]
+Customer can be deleted using:
+
+<center><b>Example curl request</b></center>
+
+    curl -H "content-type: application/json" -H "accept:application/json" -X DELETE "https://api.birdeye.com/resources/v1/customer/id/411504416?api_key=92bcd6e0-c102-43fd-8a67-1a7be5258451" –v
```

Comment:
@ameyavirkar I have used same API-key, which is used for other APIs in documents.  Also verified this is not a valid key.